pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -568,25 +563,11 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 		mounter:           volumeMounter,
 		blockVolumeMapper: volumeMapper,
 	}
+	// Remember to restore devicePath
+	rc.volumesNeedDevicePath = append(rc.volumesNeedDevicePath, uniqueVolumeName)
```

Comment:
why this is needed? how about linen 410?

Reply:
Good catch, deleted. I kept it at line 410 / `reconstructVolumes()`